packages/react/spec/auto/PolarisAutoTable.stories.jsx

```diff
@@ -182,3 +180,64 @@ export const CustomCellWithDeleteButton = {
     ],
   },
 };
+
+export const ActionParametersLol = {
+  args: {
+    model: api.autoTableTest,
+    excludeActions: ["delete"],
+    actions: [
+      "customAction",
+      {
+        label: "Sum ids",
+        callback: (ids) => {
+          const idSum = ids.reduce((i, total) => parseInt(i) + parseInt(total));
+          return idSum;
```

Comment:
What does it mean functionally for the callback to return a value like this?

Reply:
A defined callback return value is shown in the confirmation modal after the action is run. if there is no defined renderer

Reply:
I think the behavior for custom action handlers is it should override the modal behavior, it should just be a callback with the ids/records.

Reply:
you would have to implement your own confirmation/success messages as you desired.

Reply:
Do you think there should be a confirmation modal at all if you have a callback with no renderer? Otherwise it could be hard to tell if you clicked on the action or closed the menu without that feedback

Reply:
Updated `actions` such that
- Passing in a string will look for a corresponding Gadget bulk action on the model
  - Has a confirmation modal before it is run
- Passing in an object with `callback` will call upon that function with `(ids, selectedRows)` immediately upon selecting the action from the dropdown
- Passing in an object with `render` will load a modal containing the result of the `render` function. 
- `render` and `callback` are mutually exclusive. Having both or none in an object will throw an error